pertpy/plot/_guide_rna.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import logging
+
+import numpy as np
+import scanpy as sc
+from anndata import AnnData
+
+
+class GuideRnaPlot:
+    @staticmethod
+    def heatmap(
+        adata: AnnData,
+        layer: str | None = None,
+        order_by: np.ndarray | str | None = None,
+        key_to_save_order: str = None,
+        **kwds,
+    ) -> None:
+        """Simple gRNA plotting
```

Comment:
This description isn't the most useful. Please add a more descriptive one liner.